src/Microsoft.AspNet.Mvc.Core/ModelBinding/CollectionModelBinder.cs

```diff
@@ -149,12 +149,16 @@ protected object CreateInstance(Type targetType)
                 bindingContext.ModelName,
                 bindingContext.ModelMetadata,
                 boundCollection);
+
+            var innerBindingContext = ModelBindingContext.CreateChildBindingContext(
+                bindingContext,
+                elementMetadata,
+                fieldName: string.Empty,
```

Comment:
Couldn't think of anything super useful for collection model binder to put here, so it's just empty. I can't think of a scenario where it would be used.